cmd/tester/main.go

```diff
@@ -6,27 +6,56 @@ package main
 
 import (
 	"fmt"
+	"log"
+	"os"
+	"strings"
 	"time"
 
 	"github.com/openfaas-incubator/connector-sdk/types"
 )
 
 func main() {
-	creds := types.GetCredentials()
-	config := &types.ControllerConfig{
-		RebuildInterval: time.Millisecond * 1000,
-		GatewayURL:      "http://127.0.0.1:8080",
-		PrintResponse:   true,
-	}
+	creds := types.GetCredentials() // Get credentials for gateway login
+	config := getControllerConfig()
 
 	controller := types.NewController(creds, config)
 	fmt.Println(controller)
 	controller.BeginMapBuilder()
 
+	topics := getTopics()
+
 	// Simulate events emitting from queue/pub-sub
 	for {
 		time.Sleep(2 * time.Second)
```

Comment:
Make this configurable to please.

Reply:
Should I add an env-var to make this configurable ? Or a simple variable is enough ?